src/components/MarkdownDescription.tsx

```diff
@@ -32,7 +32,30 @@ export default function MarkdownDescription({description}: MarkdownDescriptionPr
         },
       }}>
       <Typography variant="body1" component="div">
-        <ReactMarkdown>{description}</ReactMarkdown>
+        <ReactMarkdown
+          allowedElements={[
```

Comment:
We may also want to include table elements like `table`, `tr`, `th`, `td`, `tbody`, `tfoot`, `thead`, and `caption`